doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -87,19 +87,29 @@ Integrated Security=True
         </ctorConnectionString>
         <ApplicationIntent>
             <summary>Declares the application workload type when connecting to a database in an SQL Server Availability Group. You can set the value of this property with <see cref="T:Microsoft.Data.SqlClient.ApplicationIntent" />. For more information about SqlClient support for Always On Availability Groups, see [SqlClient Support for High Availability, Disaster Recovery](/sql/connect/ado-net/sql/sqlclient-support-high-availability-disaster-recovery).</summary>
-            <value>Returns the current value of the property (a value of type <see cref="T:Microsoft.Data.SqlClient.ApplicationIntent" />).</value>
-            <remarks>To be added.</remarks>
+            <value>Returns the current value of the property.</value>
+             <remarks>
+            <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This property corresponds to the "Application Intent" and "ApplicationIntent" keys within the connection string.  
+
+The default value is `ApplicationIntent.ReadWrite`. 
+  
+ ]]></format>
+            </remarks>
             <related type="Article" href="/sql/connect/ado-net/overview-sqlclient-driver">Overview of the SqlClient driver</related>
         </ApplicationIntent>
         <ApplicationName>
             <summary>Gets or sets the name of the application associated with the connection string.</summary>
-            <value>The name of the application, or ".NET SqlClient Data Provider" if no name has been supplied.</value>
+            <value>The name of the application, or "Core Microsoft SqlClient Data Provider" if no name has been supplied.</value>
```

Comment:
```suggestion
            <value>The name of the application. If no name has been supplied, "Framework Microsoft SqlClient Data Provider" when running on .NET Framework and "Core Microsoft SqlClient Data Provider" otherwise.</value>
```